contribute/design/1.x/cstor-operator/doc.md

```diff
@@ -610,4 +751,4 @@ Once you have created an SPC -- you do not get enough information from the SPC e
 `StoragePoolClaim` is at cluster scope which can cause a problem for multiple instances of OpenEBS running in the same cluster in different namespaces. Idea is to enable teams to use OpenEBS in their provided namespace in the cluster
 
 
-Pool topology information is not visible on the SPC spec and it hides information about block devices/disks that constitute a cStor pool on a node. Additionally, this lack of information poses challenges in incorporating pool day 2 operations features such as pool expansion, block device replacement, etc.
\ No newline at end of file
+Pool topology information is not visible on the SPC spec and it hides information about block devices/disks that constitute a cStor pool on a node. Additionally, this lack of information poses challenges in incorporating pool day 2 operations features such as pool expansion, block device replacement, etc.
```

Comment:
Are we talking about cspc or SPC. Isn't this feature for cspc way of provisioning.

Reply:
I think here we are trying how limitations in SPC solved via CSPC(feature version of SPC)